From 66ab25ecd9cd415d605b954ff092538d0b542a46 Mon Sep 17 00:00:00 2001
From: gengzonglin <gengzonglin@eswincomputing.com>
Date: Tue, 21 Oct 2025 16:29:40 +0800
Subject: [PATCH 532/540] fix:DC can't be resume during S3.

Changelogs:
	1.Use the correct irqnum to disable the DC irq.
	2.Remove Macro CONFIG_ESWIN_MMU to fix screen flash.

Signed-off-by: gengzonglin gengzonglin@eswincomputing.com
---
 drivers/gpu/drm/eswin/es_drv.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/eswin/es_drv.c b/drivers/gpu/drm/eswin/es_drv.c
index 83998a9495c0..222792da9e68 100644
--- a/drivers/gpu/drm/eswin/es_drv.c
+++ b/drivers/gpu/drm/eswin/es_drv.c
@@ -345,6 +345,7 @@ static int es_drm_bind(struct device *dev)
 	}
 	DRM_INFO("drm dev is on die%d\n", id);
 	priv->die_id = id;
+
 	priv->mmu_constructed = false;
 
 	if (drm_dev->unique) {
-- 
2.54.0

